package.json

```diff
@@ -43,7 +43,7 @@
     "chai-subset": "^1.6.0",
     "debug": "^3.1.0",
     "globby": "^8.0.1",
-    "japa": "^1.0.6",
+    "japa": "link:.yalc/japa",
```

Comment:
I don't believe this is intended

Reply:
no sorry I accidentally did an git commit -am, should be fixed now